tests/scaletest/scale_test.go

```diff
@@ -73,7 +73,7 @@ var (
 	ingressType              string
 	numOfIng                 int
 	numOfLBSvc               int
-	numOfPodsForLBSvc        = 100
+	numOfPodsForLBSvc        = 98
```

Comment:
just a query, why is this changed to 98 from 100 ? :-)

Reply:
I had to change it back, it's from something else I was checking out previously.